src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -188,7 +188,9 @@ public ConfidentialClientApplicationBuilder WithClientAssertion(Func<string> cli
         /// use <see cref="ConfidentialClientApplication.AttemptRegionDiscovery"/> and MSAL.NET will attempt to auto-detect the region.                
         /// </param>
         /// <remarks>
-        /// Region names as per https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.resourcemanager.fluent.core.region?view=azure-dotnet.
```

Comment:
Updating the URL here which have all the regions offered by azure. The URL mentioned previously did not mention the short names that the user is supposed to provide and was confusing.

Reply:
Can you add an AKA.MS link to the internal doc as well? When we'll have this properly documented on dosc.ms we can switch it over.

Reply:
I can add the link to the MSI repo but do you know if it will be available to all 1P? I thought only RPs will have access to it

Reply:
Or you mean the link to [RegionMap.json](https://msazure.visualstudio.com/DefaultCollection/One/_git/AD-Fun-ServiceEngineering?path=%2Fsrc%2Ftools%2FAzureConfig%2Flib%2FRegionMap.json)